rotkehlchen/tests/utils/blockchain.py

```diff
@@ -362,10 +362,10 @@ def mock_requests_get(url, *args, **kwargs):
                 decoded_input = web3.codec.decode_abi(input_types, bytes.fromhex(data[10:]))
                 args = []
                 for account_address in decoded_input[0]:  # pylint: disable=unsubscriptable-object  # noqa: E501
-                    account_address = to_checksum_address(account_address)
+                    account_address = deserialize_ethereum_address(account_address)
```

Comment:
for all of those in this function use the `string_to_xxxx()` function.
Lines: 346, 365,368, 399, 402

These are test functions that mock external responses. No need to perform actual deserialization here.